app/src/main/java/testConfig/Config.kt

```diff
@@ -36,6 +36,7 @@ class Config {
         const val userName: String = "userName"
         const val avatar: String = "serverUrl/avatar/userName?format=jpeg&rc_uid=null&rc_token=null"
         const val userAvatar: String = "$currentServer/avatar/userName?format=jpeg&rc_uid=null&rc_token=null"
+        const val UPDATED_AVATAR: String = "$currentServer/avatar/$USERNAME?format=jpeg&rc_uid=null&rc_token=null"
```

Comment:
Govind, small thing.....  this file uses a mix of camelCase and ALL_CAPS.  Is there a reason for that?  Perhaps it should be uniform?

Reply:
Actually, Earlier I was thinking to make some difference in the two types of the variable(Plain string variable with a value similar to name and a variable with some info value). But It is not a good idea. 😜
I will update them.